src/main/kotlin/com/workos/webhooks/models/OrganizationMembershipDeletedEvent.kt

```diff
@@ -0,0 +1,20 @@
+package com.workos.webhooks.models
+
+import com.workos.usermanagement.models.OrganizationMembership
+
+/**
+ * Webhook Event for `organization_membership.deleted`.
+ */
+class OrganizationMembershipDeletedEvent(
+  @JvmField
+  override val id: String,
+
+  @JvmField
+  override val event: EventType,
+
+  @JvmField
+  override val data: OrganizationMembership,
+
+  @JvmField
+  override val createdAt: String
```

Comment:
Should this be `deletedAt`?

Reply:
`createdAt` represents the event's created timestamp, so would be the same for all events. I'm actually just realizing now that we probably don't need to be overriding this anywhere for WebhookEvent.